src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ListViewTests/UnoSamples_Tests.ListView.cs

```diff
@@ -167,6 +167,66 @@ public void ListView_ObservableCollection_Unused_Space()
 			Assert.AreEqual(3 * heightBefore, heightAfter);
 		}
 
+		[Test]
+		[AutoRetry]
+		public void ListView_ItemClicked_Validation()
+		{
+			Run("SamplesApp.Windows_UI_Xaml_Controls.ListView.ListView_ItemClick");
+
+			var itemClickedTextBlock = _app.Marked("ItemClickedTextBlock");
+			var numberListTextBlock = _app.Marked("NumberListTextBlock");
+
+			// Assert click on 1st item
+			Assert.AreEqual("", itemClickedTextBlock.GetDependencyPropertyValue("Text")?.ToString());
+			numberListTextBlock.AtIndex(0).Tap();
+			Assert.AreEqual("1", itemClickedTextBlock.GetDependencyPropertyValue("Text")?.ToString());
+
+			// Assert click on 4th item
+			numberListTextBlock.AtIndex(3).Tap();
+			Assert.AreEqual("4", itemClickedTextBlock.GetDependencyPropertyValue("Text")?.ToString());
+
+			// Assert click on 8th item
+			numberListTextBlock.AtIndex(7).Tap();
+			Assert.AreEqual("8", itemClickedTextBlock.GetDependencyPropertyValue("Text")?.ToString());
+		}
+
+		[Test]
+		[AutoRetry]
+		public void ListView_ChangingText_Validation()
+		{
+			Run("SamplesApp.Windows_UI_Xaml_Controls.ListView.ListView_Changing_Text");
+
+			var itemClickedTextBlock = _app.Marked("ItemClickedTextBlock");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'itemClickedTextBlock'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826515)

Reply:
@carldebilly is it possible to disable this robot? comments do not make sense in numerous cases apparently

Reply:
@manpreetkaurnventive yes can be removed since not used for this test

Reply:
removed